packages/common/test-configs/cypress.json

```diff
@@ -0,0 +1,5 @@
+{
+  "baseUrl": "http://localhost:9000",
+  "video": false,
+  "defaultCommandTimeout": 5000
+}
```

Comment:
Shall we move `cypress.json` into `cypress` folder?

Reply:
If we put `cypress.json` in `cypress` folder we may need to explicitly set `integrationFolder` in the configuration file

Reply:
@florkbr had suggested having the configurations in the `packages/common` directory where all the test configurations are. What are your thoughts Bryan?
Cypress would normally look for `cypress.json` by default in the root directory from where we trigger the cypress run. @yapei any particular reason for having it within the `cypress` directory?

Reply:
Ah yeah, you are right @vidyanambiar `cypress.json` should be in root directory where we trigger the tests, it still make sense if we have other considerations

Reply:
@yapei Is your recommendation to move it into the root directory then? 
FYI @florkbr

Reply:
@vidyanambiar  I'm ok to leave it in `packages/common` if you have talked about it : )